<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BolusWindow</class>
 <widget class="QWidget" name="BolusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bolus Window</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Glucose Level (mmol/L):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="glucoseLevel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>34</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Carb Intake (g):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="carbIntake">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Correction Factor (unit/mmol):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="correctionFactor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>106</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="extendedBolus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>127</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extended Bolus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="quickBolus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>102</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick Bolus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="standardBolus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Glucose Level (mmol/L):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ICR (unit/g):</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetBGLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="carbRatioLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="extendedTimeInput">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time (hours):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
